packages/features/bookings/Booker/config.ts

```diff
@@ -50,42 +50,60 @@ export const resizeAnimationConfig: ResizeAnimationConfig = {
       minHeight: "0px",
       gridTemplateAreas: `
           "meta"
+          "header"
           "main"
           "timeslots"
         `,
```

Comment:
So header now only displays the 24h selector on mobile? and not the layout view?